main.go

```diff
@@ -76,6 +76,17 @@ type Config struct {
 }
 
 func stage2(cli *cli.Context) (int, error) {
+	mounts, err := mount.GetMounts()
+	if err != nil {
+		return -1, err
+	}
+	for _, m := range mounts {
+		if m.Mountpoint == "/" && strings.HasPrefix(m.Optional, "shared") {
```

Comment:
i think this will have the format "tag:value tag:value".  I think you want to split (strings.Fields) and look for "shared:" prefix

Reply:
actually you need to after spliting the fields split on ":" and make sure the first part equals "shared"

Reply:
addressed